synadm/cli/__init__.py

```diff
@@ -376,10 +380,15 @@ def root(ctx, verbose, batch, output, config_file):
     default value 'auto-retrieval' will try to discover the name using the
     method set by --server-discovery."""
 )
+@click.option(
+    "--ssl_verify", "-n",  default=True,
+    help="""Verify certificate (Default True, False allow to manage \
+        selfsigned certificate)."""
+)
```

Comment:
Default true can be specified by Click's show_default=True flag. Use that and I think we could then rephares this help text to something like: "Whether or not SSL certificates should be verified. Set to False to allow self-signed certifcates." or maybe even shorter because ssl_verify tells it all to a regular Matrix user/admin already:

Set to False to allow self-signed certifcates.

Reply:
Yaml seems to support different case: |true|True|TRUE|false|False|FALSE
According to https://yaml.org/type/bool.html